contracts/Liquidator/Liquidator.sol

```diff
@@ -318,12 +392,45 @@ contract Liquidator is Ownable2StepUpgradeable, ReentrancyGuardUpgradeable, Liqu
         if (!vTokenCollateral.transfer(msg.sender, theirs)) {
             revert VTokenTransferFailed(address(this), msg.sender, theirs);
         }
-        if (!vTokenCollateral.transfer(treasury, ours)) {
-            revert VTokenTransferFailed(address(this), treasury, ours);
+
+        if (IVToken(address(vTokenCollateral)).redeem(ours) != 0) {
+            pendingRedeem.push(address(vTokenCollateral));
+        } else {
+            if (address(vTokenCollateral) == address(vBnb)) {
+                _reduceBnbReserves();
+            } else {
+                _reduceVTokenReserves(address(vTokenCollateral));
+            }
         }
         return (ours, theirs);
     }
 
+    function _reduceBnbReserves() private {
+        uint256 bnbBalance = address(this).balance;
+        IWBNB(wBNB).deposit{ value: bnbBalance }();
+        if (!IWBNB(wBNB).transfer(protocolShareReserve, bnbBalance)) {
+            revert UnderlyingTransferFailed(address(vBnb), wBNB);
+        }
```

Comment:
Maybe just
```suggestion
        IERC20Upgradeable(wBNB).safeTransfer(protocolShareReserve, bnbBalance);
```

Reply:
To avoid any confusion I think we can use transfer for `wBNB` transfers.....also `wBNB` [contract](https://bscscan.com/address/0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c#code) does not have `safeTransfer` WDYT?

Reply:
But WBNB is ERC-20, so it should be possible to use SafeERC20 wrappers. I don't see any risk with `transfer`, but it would trigger automated analysis tools, so I'd use oz's wrapper to unify the approach.